index.html

```diff
@@ -24,7 +24,10 @@
                 .replace(/</g, "&lt;")
                 .replace(/>/g, "&gt;")
                 .replace(/"/g, "&quot;")
-                .replace(/'/g, "&#039;");
+                .replace(/'/g, "&#039;")
```

Comment:
This changes how the tracks are _displayed_ but doesn't address the issue at all. The parser needs to replace these. You will also need to include unit tests.

Here is a similar diff: https://github.com/elliotchance/tracklist-editor/pull/2